fedimint-client-rpc/src/lib.rs

```diff
@@ -436,6 +437,10 @@ impl RpcGlobalState {
                 let words = self.get_mnemonic_words().await?;
                 yield serde_json::json!({ "mnemonic": words });
             })),
+            RpcRequestKind::HasMnemonicSet => Some(Box::pin(try_stream! {
+                let is_set = self.has_mnemonic_set().await?;
+                yield serde_json::json!({ "is_set": is_set });
+            })),
```

Comment:
I absolutely hate this api style response


```
yield json!(is_set)
```

same for above ones what is `{ success: true }` just do `json!(())` there is a separate highway for errors anyways.